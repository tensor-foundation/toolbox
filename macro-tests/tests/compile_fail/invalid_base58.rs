use solana_program::pubkey::Pubkey;
use solana_pubkey_macro::pubkey;

fn main() {
    let _pubkey: Pubkey = pubkey!("Not a valid base58 string"); // Should fail: invalid base58
}
