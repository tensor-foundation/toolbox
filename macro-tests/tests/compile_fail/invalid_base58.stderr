error[E0432]: unresolved import `solana_pubkey_macro`
 --> tests/compile_fail/invalid_base58.rs:2:5
  |
2 | use solana_pubkey_macro::pubkey;
  |     ^^^^^^^^^^^^^^^^^^^ use of undeclared crate or module `solana_pubkey_macro`
